homework-submission/index.js

```diff
@@ -0,0 +1,75 @@
+/*to create a list item <li> element whose props are description and deadline*/
+
+const ListItem = ({ description, deadline, className }) => {
+  return (
+    <li className={className}>
+      Description: {description} Deadline: {deadline}
+    </li>
+  );
+};
+
+const StaticList = () => {
+  return (
+    <ul>
+      <p>This is the Static List</p>
+      <ListItem description="Get out of bed" deadline="2017-09-11" className="strike_through" />
+      <ListItem description="Brush teeth" deadline="Brush teeth" className="" />
+      <ListItem description="Eat breakfast" deadline="2017-09-09" className="" />
+    </ul>
+  );
+};
+
+const DynamicList = () => {
+  const dynamicData = [
+    {
+      id: 1,
+      description: 'Get out of bed',
+      deadline: '2017-09-11',
+      done: true,
+    },
+    {
+      id: 2,
+      description: 'Brush teeth',
+      deadline: 'Brush teeth',
+      done: false,
+    },
+    {
+      id: 3,
+      description: 'Eat breakfast',
+      deadline: '2017-09-09',
+      done: false,
+    },
+  ];
+  return (
+    <ul>
+      <p>This is the Dynamic List</p>
+      {dynamicData.map(({ id, description, deadline, done }) => {
+        return (
+          <ListItem
+            key={id}
+            description={description}
+            deadline={deadline}
+            className={done === true ? 'strike_through' : ''}
+          />
+        );
+      })}
+    </ul>
+  );
+};
+
+const Container_List = () => {
+  return (
+    <div>
+      <StaticList />
+      <DynamicList />
+    </div>
+  );
+};
+
+class ContainerList extends React.Component {
+  render() {
+    return <Container_List />;
```

Comment:
I confused 😕
you have both names `Container_List` and `ContainerList` which makes it a big deal for you in larger applications.

if no need for this Component, you can remove it, but just no need for making similar names.

Reply:
I deleted unnecessary Container_List variable and write down dynamic and static list directly into the ContainerList. Less code better readibility!